src/ontology/mondo-edit.obo

```diff
@@ -410465,20 +410465,22 @@ relationship: has_characteristic MONDO:0021152 {source="OMIMPS:277600"} ! inheri
 
 [Term]
 id: MONDO:0018097
-name: West syndrome
-def: "West syndrome (or infantile spasms) is characterized by the association of clusters of axial spasms, psychomotor retardation and an hypsarrhythmic interictal EEG pattern. It is the most frequent type of epileptic encephalopathy. It may occur in otherwise healthy infants and in those with abnormal cognitive development." [Orphanet:3451]
-comment: Editor note: check placement of OMIM IDs and EIEE subtypes
+name: infantile spasms
+def: "A rare epilepsy syndrome characterized by onset of epileptic spasms in infants between 2 and 12 months of age, and rarely up to 24 months. Infants may have no antecedent history, or a history reflecting the underlying cause. The classical triad of epileptic spasms, hypsarrhythmia and developmental stagnation or regression is historically referred to as West syndrome." [Orphanet:3451]
 subset: gard_rare {source="GARD:7887", source="MONDO:GARD"}
 subset: nord_rare {source="NORD:1848", source="MONDO:NORD"}
 subset: ordo_clinical_syndrome {source="Orphanet:3451"}
 subset: ordo_disorder {source="Orphanet:3451"}
 subset: orphanet_rare {source="Orphanet:3451"}
 subset: otar {source="MONDO:OTAR"}
 subset: rare
+synonym: "IESS" EXACT ABBREVIATION [https://www.epilepsydiagnosis.org/syndrome/west-syndrome-overview.html]
+synonym: "infantile epileptic spasms syndrome" EXACT [https://www.epilepsydiagnosis.org/syndrome/west-syndrome-overview.html]
 synonym: "infantile spasms" EXACT [icd11.foundation:1023597213]
-synonym: "Infantile spasms syndrome" RELATED [DOID:0050562]
+synonym: "infantile spasms syndrome" RELATED [DOID:0050562]
```

Comment:
@nicolevasilevsky I think this synonym should be exact.

Reply:
done!